### ISSUE TITLE
Default state testing requires action if logic set

### ISSUE THREAD

**Issue by erkiesken** [jeffbski/redux-logic-test#2]
Thanks for this lib, really useful and straighforward to use.

But when testing for initial store state I ran into an issue. Using Jest and this code:
```
describe("app reducer", () => {
    let store;
    beforeEach(() => {
        store = createMockStore({
            reducer,
            logic,
        });
    });
    it("(failing) returns default state", (done) => {
        store.whenComplete(() => {
            expect(store.getState()).toEqual(AppStoreState({}));
            done();
        });
    });
});
```

It will fail because `whenComplete` is never called. But if i remove `logic` from the mock store, this test succeeds.

As workaround I'm dispatching the special `@@redux/INIT` for now:
```
    it("returns default state", (done) => {
        store.dispatch({ type: "@@redux/INIT" });
        store.whenComplete(() => {
            expect(store.getState()).toEqual(AppStoreState({}));
            done();
        });
    });
```

Not sure if this is a bug or just a gotcha in the implementation that needs documenting?

PS: Reducer itself is pretty vanilla, ie:
```
function getDefaultState() {
    return AppStoreState({}); // AppStoreState is a custom tcomb struct type
}

export default function reducer(state = getDefaultState(), action) {
    switch (action.type) {
...
```

**Comment by jeffbski** [comment]
@tehnomaag Thanks for mentioning this. I'll take a closer look. I hadn't really looked at the situation where nothing was dispatched and then calling whenComplete() so let me see if I can correct this, it should probably just complete like you expected.

**Comment by jeffbski** [comment]
This ended up being a bug in redux-logic so I have corrected it and referenced the newer version redux-logic@0.11.8

If nothing has been dispatched, then whenComplete should still complete.

Update your redux-logic-test and redux-logic to test this.

Published in redux-logic-test@1.0.4 redux-logic@0.11.8

**Comment by jeffbski** [comment]
The fix should close this, but let me know if you see any other problems.
